gradle.properties

```diff
@@ -1,4 +1,4 @@
-VERSION_NAME=1.8.31-DTREE-6-SNAPSHOT
+VERSION_NAME=1.8.31-DTREE-8-ALPHA-SNAPSHOT
```

Comment:
Is the versioning name intentional here

Reply:
Yes, the current version is 7 but then I thought maybe we should start doing alpha, beta, gamma to avoid going up version for small changes @issyzac